packages/core/src/transport/transport.ts

```diff
@@ -30,7 +30,9 @@ export class HttpRequest {
 }
 
 function addBatchTime(url: string) {
-  return `${url}${url.indexOf('?') === -1 ? '?' : '&'}batch_time=${new Date().getTime()}`
+  return `${url}${url.indexOf('?') === -1 ? '?' : '&'}batch_time=${new Date().getTime()}&m_time=${getTimestamp(
```

Comment:
I guess `m` stands for `monotonic`?

Reply:
yep, not sure we need too bee too explicit about it